.gitignore

```diff
@@ -0,0 +1,28 @@
+*.o
+*.obj
+*.a
+*.so
+*.out.*
+*.err.*
+*.err
+*.out
+config.log
+config.status
+Makefile.config
+HYPRE_config.h
+autom4te.cache
+*.TVD.v4breakpoints*
+
+# Drivers #
```

Comment:
The gitignore file should be set up to match master, so we need to decide whether we should have these entries in master also.  There was some discussion about gitignore in one of the other issues or PRs.

Reply:
FYI, the discussion about gitignore was in PR #243 .  As a strategy, we probably do want to add anything that might potentially be committed by accident by developers, so adding the driver executables probably makes sense.  On the other hand, this could get out of hand if you consider things like the AUTOTEST directory.  I think I will create an issue to have a short discussion and settle on a convention.

Reply:
Thanks for pointing this out, Rob! I have the drivers executables in my local gitignore and thought it could be useful to have it in the repo as well. The same for the TVD file which is created by totalview.